why-oc.md

```diff
@@ -55,7 +55,7 @@ The majority of Clover's functionality is actually supported in OpenCore in the
 * Does not support hiding specific drives
 * Does not support changing settings within OpenCore's menu
 * Does not patch PCIRoot UID value
-* Does not support macOS-only ACPI injection and patching
```

Comment:
That's not true. OpenCore, or better its ACPI injection mechanism supports macOS-only ACPI injection.
That's the case of `If (_OSI("Darwin")) {...}` as used in `SSDT-EC-USBX` (as well as the good old deprecated `SSDT-RHUB`; check my previous comment for more explanations)